### PR TITLE
Iow 666 approval level

### PR DESCRIPTION
**Pull request by ee-usgs**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [x] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

File: src/test/java/gov/usgs/wma/waterdata/groundwater/DiscreteGroundWaterRulesTest.java [hunk 1]
```diff
@@ -23,17 +14,21 @@
...
+		dgw.approvalLevel = "1200";
 	}
 
 
+	//
+	//Rule:  The AQ measurement reading qualifiers (multi-values) are mapped to a single NWISWeb lev_status_cd.
+
 	@Test
-	void singleRecognizedValueIsMapped() throws Exception {
+	void qualifiersSingleRecognizedValueIsMapped() throws Exception {
```
Comment (ee-usgs) [inline src/test/java/gov/usgs/wma/waterdata/groundwater/DiscreteGroundWaterRulesTest.java:25]: I renamed all the qualifier tests to separate them from the approval tests.  In the future, I could see separating individual rules to individual classes, but so far its not too unweildly...

File: src/test/java/gov/usgs/wma/waterdata/groundwater/RdbWriterTest.java [hunk 2]
```diff
@@ -57,7 +53,7 @@ DiscreteGroundWater makeDgw() {
 		dgw.dateTimeAccuracyCode = "D"; // [D]day or [M]minute
 		dgw.timezoneCode = "UTC";
 		dgw.timeMeasuredUtc = "01-MAY-2007 12:00:00"; // UTC
-		dgw.approvalStatusCode = "T"; // T or R
```
Comment (ee-usgs) [inline src/test/java/gov/usgs/wma/waterdata/groundwater/RdbWriterTest.java:60]: I'm not sure where the "T" or "R" concept came from in the tests, but the actual approval values have never before made it from the AQTS data to the rdb file, so the assumption in the test file here doesn't really matter.

File: src/main/java/gov/usgs/wma/waterdata/groundwater/DiscreteGroundWaterRules.java [hunk 3]
```diff
@@ -76,6 +76,23 @@ public void apply(DiscreteGroundWater domObj) {
 		}
 
 
+		//Rule:  Only the Aquarius TS "1200" approvalLevel is considered approved.  All others values or no value is
+		// considered provisional.  "A" is the NWISWeb code for Approved, "P" for Provisional (ie not approved).
+		//If the approvalLevel is not recognized as valid, it is ignored and the value considered provisional.
+		//Ref:  https://internal.cida.usgs.gov/jira/browse/IOW-666
```
Comment (ssoper-usgs) [inline src/main/java/gov/usgs/wma/waterdata/groundwater/DiscreteGroundWaterRules.java:82]: Maybe also add the URL to nwis approval codes

**Review by ssoper-usgs — APPROVED** [review]
Looks good!

Note the travis-ci build will fail until the observations-db liquibase changes are merged, and the docker image on dockerhub is updated so that travis is pulling your updated version of the table.

**Review by orosz-usgs — APPROVED** [review]

**Pull request merged**